docs/guide/caching.md

```diff
@@ -1,51 +1,40 @@
 # Caching
 
-You can cache your API responses to improve performance between multiple calls like for page navigation.
+If your endpoint supports [client caching](https://developer.mozilla.org/en-US/docs/Web/HTTP/Guides/Caching), the response will be cached by the browser. Subsequent GET requests to the same endpoint will return the cached response, if available, without making a new request to the server.
```

Comment:
With the proxy request workaround – will the caching work for POST requests as well?

Reply:
I get that, but I fear this will make Nuxt API Party useless for some devs. I have at least two client projects where all requests to the API are POST requests, because of the query language used. But specific requests (even tho they are POST requests) shall be cached...

Do you see a way to combine the native approach with a caching layer just like the current version of Nuxt API Party uses for this particular usecase?

Reply:
I understand. I also have an endpoint which is all POST via RPC.